data/wacom.stylus

```diff
@@ -91,162 +66,89 @@ Type=Mobile
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Pressure
-Type=Mobile
+AliasOf=0x1
 
 [0x271]
 # Wacom  ; VID_NONE     | 0x0000 | BAT_SWAP ("Wacom Bamboo Ink" / CS323A)
-Name=Bamboo Ink
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Pressure
-Type=Mobile
+AliasOf=0x49
 
 [0x421]
 # HP     ; VID_NONE     | 0x0000 | BAT_SWAP
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Pressure
-Type=Mobile
+AliasOf=0x1
 
 [0x431]
 # Dell   ; VID_BROADCOM | 0x81B9 | BAT_SWAP
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Pressure
-Type=Mobile
+AliasOf=0x1
 
 [0x621]
 # Lenovo ; VID_NONE     | 0x0000 | BAT_SWAP
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Pressure
-Type=Mobile
+AliasOf=0x1
 
 [0x631]
 # Dell   ; VID_NONE     | 0x0000 | BAT_SWAP | LONGPRESS (Dell PN557W)
 # Dell   ; VID_NONE     | 0xf142541e | BAT_SWAP (Dell PN5122W)
 Name=Dell Active Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Pressure
-Type=Mobile
+AliasOf=0x1
 
 [0x8051]
 # Google ; VID_NONE     | 0x0000 | BAT_SWAP ("Google Pixelbook Pen" / C0B)
-Name=AES Pen
-Group=isdv4-aes
+AliasOf=0x1
 Buttons=0
-EraserType=Button
 Axes=Tilt;Pressure
-Type=Mobile
 
 [0x805B]
 # Dell   ; VID_BROADCOM | 0x81D5 | BAT_SWAP | BAT_GATT | BAT_SHARED | LONGPRESS (Dell PN579X)
 # Lenovo ; VID_LENOVO   | 0x60C5 | BAT_SWAP | BAT_GATT | BAT_SHARED
 # Toshiba; VID_NONE     | 0x0000 | BAT_SWAP
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Tilt;Pressure
-Type=Mobile
+AliasOf=0x1
 
 [0x806B]
 # HP     ; VID_CHICONY  | 0x1728 | BAT_CHRG | BAT_PROX | BAT_SHARED | LONGPRESS | PROX ("HP Rechargeable Active Pen" / HP Active Pen G2 / 4KL69AA)
 # Huawei ; VID_NONE     | 0x0000 | BAT_SWAP
 # Lenovo ; VID_LENOVO   | 0x60C2 | BAT_CHRG | BAT_GATT | BAT_SHARED
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Tilt;Pressure
-Type=Mobile
+AliasOf=0x1
 
 [0x807B]
 # Wacom  ; VID_WACOM    | 0x0397 | BAT_CHRG | BAT_GATT | BAT_SHARED | LONGPRESS ("Wacom Bamboo Ink Plus" / CS322A)
+AliasOf=0x49
 Name=Bamboo Ink Plus
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
 Axes=Tilt;Pressure
-Type=Mobile
 
 [0x826B]
 # HP     ; VID_CHICONY  | 0x1728 | BAT_CHRG | BAT_PROX | BAT_SHARED | LONGPRESS | PROX ("HP Active Pen G3" / L08263-003 / L57042-001)
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
+AliasOf=0x1
 Axes=Tilt;Pressure
-Type=Mobile
 
 [0x846B]
 # HP     ; VID_CHICONY  | 0x1850 | BAT_CHRG | BAT_GATT | BAT_SHARED | LONGPRESS | PROX ("HP Rechargeable Active Pen G3" / HP Active Pen G3 / 6SG43AA)
-Name=AES Pen
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
+AliasOf=0x1
 Axes=Tilt;Pressure
-Type=Mobile
 
 [0x219]
 # Lenovo ; VID_LENOVO     | 0x219 | BAT_SWAP
 Name=Lenovo Digital Pen 2
-Group=isdv4-aes
-Buttons=1
-EraserType=Button
-Axes=Pressure
-Type=Mobile
+AliasOf=0x1
 
 # Inking pen have no eraser
 [0x812]
 # Intuos and Intuos2
-Name=Inking Pen
+AliasOf=0x20802
```

Comment:
Is this really worth changing these? The I2 is now old enough that we should never have to think about it again so leaving this code untouched forever is better. This applies to a whole bunch of others too, IMO using aliases is best for where it's really "same thing, different id" even though the alias implementation lets us almost use it like CSS.